### PR TITLE
set, view, wait, write: add link

### PR DESCRIPTION
**Pull request by git-em**
Add "more information" links

- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

Related to: 
- #6062

### CODE REVIEW

**Comment by git-em** [comment]
#6062

File: pages/common/set.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # set
 
 > Display, set or unset values of shell attributes and positional parameters.
+> More information: <https://www.unix.com/man-page/Linux/1/set>.
```
Comment (mfrw) [inline pages/common/set.md:4]:
```suggestion
> More information: <https://manned.org/set>.
```

File: pages/common/wait.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # wait
 
 > Wait for a process to complete before proceeding.
+> More information: <https://www.unix.com/man-page/Linux/1/wait>.
```
Comment (mfrw) [inline pages/common/wait.md:4]:
```suggestion
> More information: <https://manned.org/wait>.
```

File: pages/common/write.md [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 > Write a message on the terminal of a specified logged in user (ctrl-C to stop writing messages).
 > Use the `who` command to find out all terminal_ids of all active users active on the system. See also `mesg`.
+> More information: <https://www.unix.com/man-page/Linux/1/write>.
```
Comment (mfrw) [inline pages/common/write.md:5]:
```suggestion
> More information: <https://manned.org/write>.
```

File: pages/common/view.md [hunk 4]
```diff
@@ -2,6 +2,7 @@
 
 > A read-only version of `vim`.
 > This is equivalent to `vim -R`.
+> More information: <https://www.vim.org>.
```
Comment (mfrw) [inline pages/common/view.md:5]:
```suggestion
> More information: <https://www.vim.org/docs>.
```

**Review by mfrw — COMMENTED** [review]
Thank you for your contribution @git-em.
We prefer `manned.org` for more information :)

File: pages/common/view.md (hunk 4 above)
Comment (navarroaxel) [reply pages/common/view.md:5]:
```suggestion
> More information: <https://manned.org/view>.
```

Comment (git-em) [reply pages/common/view.md:5]: I've read somewhere about favoring the authors' information pages over others. Isn't this the case @navarroaxel?

Comment (git-em) [reply pages/common/view.md:5]: By the way, thank you all for being so supportive! (:

Comment (navarroaxel) [reply pages/common/view.md:5]: Yeah, our first option it's the author's website, else `manned.org` is the first option.

Comment (git-em) [reply pages/common/view.md:5]: So which suggestion should i commit? 
I am confused because to me https://vim.org is the author's website😭

Comment (navarroaxel) [reply pages/common/view.md:5]: Not really, In Arch Linux `view` is part of the traditional `vi` package, it isn't installed with `vim`.
In Ubuntu, and Fedora: `view` it's just an alias for the same `vim`, and the package `vi` doesn't exists.
Ok, leave as-is.

**Review by navarroaxel — APPROVED** [review]

**Review by mfrw — APPROVED** [review]
Thank you @git-em .
LGTM

File: pages/common/view.md (hunk 4 above)
Comment (mfrw) [reply pages/common/view.md:5]: I am in favor of keeping `vim.org`

**Pull request merged**